### PR TITLE
Make NT Stale Data more discoverable

### PR DESCRIPTION
**Pull request by sciencewhiz**
Move from a sub-section in SmartDashboard into NetworkTables, and add references in each dashboard

### CODE REVIEW

File: source/docs/software/networktables/stale-data.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Stale Data
+==========
+
+NetworkTables acts as a distributed table of name and value pairs. If a name/value pair is added to either the server (robot) or any client (dashboard, co-processor, limelight, etc) it is replicated to all devices. If a name/value pair is deleted from, say, the robot but any NetworkTables clients are still running, then when the robot is restarted, the old values will still appear in the client because they never stopped running and continue to have those values in their tables. When the robot restarts, those old values will be replicated to the robot.
```
Comment (PeterJohnson) [inline source/docs/software/networktables/stale-data.rst:4]: This is more nuanced than stated.  NT4 has different categories of topic value preservation: persistent, retained, and transient.  Only persistent and retained topic values are preserved by the server when all publishers are gone, and only persistent topic values are preserved through a server reboot. As NT4 defaults to transient publishes, and a client disconnect acts as an “unpublish all”, stale values are rare unless a client specifically marks a topic as retained.

NT3 clients mark all their published topics as retained, so there’s a greater likelihood those values will be stale.  But this still only applies to topics the NT3 client actually writes to.

Comment (PeterJohnson) [reply source/docs/software/networktables/stale-data.rst:4]: It’s probably also worth noting here that the timestamp column in OutlineViewer can be used to see what topics are being regularly updated.

Comment (jasondaming) [reply source/docs/software/networktables/stale-data.rst:4]: I agree it is more nuanced than written but I don't think its much.

So with Limelight for example (presumably still on NT3? will never know till (if?) we get a new image) would leave all of its stale values?  I am just thinking of one of the most common use cases for teams.

File: source/docs/software/networktables/stale-data.rst [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+To ensure that the robot and clients are showing current values, it is necessary to shut down all NetworkTables clients and robot at the same time. That way, old values that one is holding won't get replicated to the others. Once everything is shut down, the robot and clients can be restarted.
```
Comment (jasondaming) [inline source/docs/software/networktables/stale-data.rst:6]:
```suggestion
To ensure that the robot and clients are showing current values, it is necessary to shut down all NetworkTables clients and the robot at the same time. That way, old values one is holding won't get replicated to the others. Once everything is shut down, the robot and clients can be restarted.
```

File: source/docs/software/networktables/stale-data.rst (hunk 1 above)
Comment (sciencewhiz) [reply source/docs/software/networktables/stale-data.rst:4]: However, Limelight isn't changing the data that it sends often. like changing robot programs

**Comment by sciencewhiz** [comment]
Since this is only really an issue with NT3, going to close this and keep in SD

**Pull request closed without merging**